stock-filters/ChirpVillage/Builder.py

```diff
@@ -0,0 +1,277 @@
+import utilityFunctions
+import RandUtils
+from Biomes import BlockUtils
+
+
+"""Utils for buildings"""
+
+def construct_walls(level, coords, biome, height):
+    minx = coords[0][0]
```

Comment:
you do this a lot. Might be worth putting it in a helper function? just to reduce reused code

Reply:
@DelightedWaif

Reply:
^ i agree

Reply:
Something like this 
`def get_coord_vals(coords):
   return coords[0][0], coords[0][1], coords[0][2], coords[1][0], coords[1][1]`
and use it like
`minx, minz, miny, maxx, maxz = get_coord_vals(coords)`